--- emesene/data/share/applications/emesene.desktop	2012-04-13 14:43:29.264139803 +0200
+++ emesene/data/share/applications/emesene.desktop.old	2012-04-13 14:43:45.639141395 +0200
@@ -7,9 +7,3 @@ Icon=emesene
 Terminal=false
 Type=Application
 Categories=Network;InstantMessaging;
-X-Ayatana-Desktop-Shortcuts=NewInstance;
-
-[NewInstance Shortcut Group]
-Name=Open a new instance
-Exec=emesene
-TargetEnvironment=Unity
